server/index.js

```diff
@@ -37,84 +37,44 @@ app.use(express.static(PUBLIC_PATH));
 app.use(express.static(DIST_PATH));
 
 io.on('connection', socket => {
+  Axios.put(`http://localhost:3000/api/user/socketConnect/${socket.id}`);
```

Comment:
we can't hardcode localhost 3000

Reply:
Yep we'll have to change it in the future